h2/src/test/org/h2/test/unit/TestIgnoreCatalogs.java

```diff
@@ -0,0 +1,158 @@
+package org.h2.test.unit;
```

Comment:
You need to add the standard copyright header (copy it from some other file).

This test case should be in the `test.db` package, I guess or in some other reasonable package.

Reply:
done